.github/workflows/native-dependencies.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Use Node.js v22.15.1
         uses: actions/setup-node@1a4442cacd436585916779262731d5b162bc6ec7 # v3.8.2
         with:
-          node-version: "22.15.1"
+          node-version: "22.17.0"
```

Comment:
We should also update this in the REMOTE_NODE_VERSION and the gitpod.Dockerfile. Searching for 22.15 in the repo should be enough.

Is this a hard requirement? if yes we should also update the engines in the package.json, which still points to 18.

Reply:
For this GH action it is a hard requirement as otherwise the remote features will not work.

Reply:
> if yes we should also update the engines in the package.json, which still points to 18.

The engine in the package.json is more of a sanity check. However I can also upgrade it because we require Node 22.12+ anyway because of `node-abi`